h2o-core/src/main/java/water/api/schemas3/FrameChunksV3.java

```diff
@@ -5,9 +5,9 @@
 import water.api.API;
 import water.fvec.Vec;
 
-public class FrameChunksV3 extends SchemaV3<Frame, FrameChunksV3> {
+public class FrameChunksV3 extends SchemaV3<Iced, FrameChunksV3> {
```

Comment:
I think this is a bit unusual but I don't see any issues with it

Reply:
There is a problem that `Frame` doesn't have a parameterless constructor and [this line](https://github.com/h2oai/h2o-3/blob/b9b0954b45c956d6d3d08537ada3543c85efa557/h2o-core/src/main/java/water/api/Schema.java#L167) will cause H2O node failure. I guess that's the reason why the `Frames` class was introduced.

Regarding the usage of `Iced`, I've gotten inspired in [`FindV3`](https://github.com/h2oai/h2o-3/blob/43f8ab952a69a8bc9484bd0ffac909b6e3e820ca/h2o-core/src/main/java/water/api/schemas3/FindV3.java#L7).